lib/kamal/cli/traefik.rb

```diff
@@ -11,20 +11,23 @@ def boot
 
   desc "reboot", "Reboot Traefik on servers (stop container, remove container, start new container)"
   option :rolling, type: :boolean, default: false, desc: "Reboot traefik on hosts in sequence, rather than in parallel"
+  option :confirmed, aliases: "-y", type: :boolean, default: false, desc: "Proceed without confirmation question"
   def reboot
-    mutating do
-      host_groups = options[:rolling] ? KAMAL.traefik_hosts : [ KAMAL.traefik_hosts ]
-      host_groups.each do |hosts|
-        host_list = Array(hosts).join(",")
-        run_hook "pre-traefik-reboot", hosts: host_list
-        on(hosts) do
-          execute *KAMAL.auditor.record("Rebooted traefik"), verbosity: :debug
-          execute *KAMAL.registry.login
-          execute *KAMAL.traefik.stop, raise_on_non_zero_exit: false
-          execute *KAMAL.traefik.remove_container
-          execute *KAMAL.traefik.run
+    confirming "Reboot Traefik on hosts? (stop container, remove container, start new container). This will cause a brief outage." do
```

Comment:
I think this is not necessarily true. If the rolling option is used and there are multiple hosts the app will not crash, only one host will become unavailable for a short time.